<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vehicle Counting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>981</width>
      <height>641</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_setting">
     <attribute name="title">
      <string>1. Setting</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_1VideoInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>411</width>
        <height>91</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>1. Video Input</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="comboBox_choseDevice">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>55</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_choseFile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_choseDevice">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Camera Port</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_selectFile">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>26</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>. . . </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_selectFile">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2VideoMode">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>411</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>2. Video Mode</string>
      </property>
      <widget class="QCheckBox" name="checkBox_showboundaryVM">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show boundary</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_gsVM">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_hsvVM">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>HSV</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_edVM">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Edge Detection</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_rgbVM">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>102</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Normal (RGB)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_showroiVM">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show region of interest (ROI)</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3DataInput">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>411</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>3. Data Input</string>
      </property>
      <widget class="QGroupBox" name="groupBox_datainputCamera">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Camera</string>
       </property>
       <widget class="QLabel" name="label_altitude">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Altitude from street</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_altitudeCam">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>26</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_altMeter">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>meter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_elevated">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>55</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Elevated from street</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_elevatedCam">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>51</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_elvDegree">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>55</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>degree</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_fps">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Frame per second</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_fpersecond">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>80</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>fps</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_fps">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>77</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_dataInputSpeedCam">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>381</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Speed Detection</string>
       </property>
       <widget class="QRadioButton" name="radioButton_PinHoleModel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pin Hole Model</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_EuclideanModel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Euclidean Model</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_dataInputThreshold">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>391</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Threshold</string>
       </property>
       <widget class="QLabel" name="label_initBackgroundSecond">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>second</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_initBackground">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>26</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_initBackground">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Initiation background</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_setSetting">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>570</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_helpSetting">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>570</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_video">
     <attribute name="title">
      <string>2. Video</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_stopVideo">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>570</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_startVideo">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>570</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pauseVideo">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>570</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_ShortVehicle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>560</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_ShortVehicle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Short Vehicle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_LongVehicle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>580</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Long Vehicle</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_LongVehicle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>580</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="video_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>971</width>
        <height>541</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_showLog">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>570</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_videoFps">
      <property name="geometry">
       <rect>
        <x>825</x>
        <y>580</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_log">
     <attribute name="title">
      <string>3. Log</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_searchLog">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>811</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Search</string>
      </property>
      <widget class="QLabel" name="label_date">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>56</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_until">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>27</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_searchLog">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>21</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clearLog">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>21</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_fromDate">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>22</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>13</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>d MMMM yyyy</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_untilDate">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>22</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>13</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>d MMMM yyyy</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logLongVehicle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>580</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Long Vehicle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logShortVehicle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Short Vehicle</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_logLongVehicle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>580</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_logShortVehicle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>560</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_searchLog">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>911</width>
        <height>451</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_settings"/>
    <addaction name="actionSave_settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_settings">
   <property name="text">
    <string>Open settings</string>
   </property>
  </action>
  <action name="actionSave_settings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Applicctions help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_startVideo</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
